packages/opentelemetry-exporter-collector/src/platform/node/util.ts

```diff
@@ -49,6 +49,14 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
   };
 
   const request = parsedUrl.protocol === 'http:' ? http.request : https.request;
+  const Agent = parsedUrl.protocol === 'http:' ? http.Agent : https.Agent;
+  if (collector.keepAlive) {
```

Comment:
if keepAlive is false and user provided `httpAgentOptions` in config I would show warning (not error) with explanation that `httpAgentOptions` is only available when keepAlive is true, WDYT ? Once only in constructor

Reply:
Makes sense, I will do that.